client/app/config/routes.js

```diff
@@ -0,0 +1,18 @@
+import HomeView from '../views/HomeView';
+import TestView from '../views/TestView';
+import TabBarView from '../views/TabBarView';
+import TemplateView from '../views/TemplateView';
+
+const TabBarRoutes = {
```

Comment:
I'm assuming that these will be changed as we build out the respective pages?

Reply:
Absolutely.